step-templates/deploy-child-project.json

```diff
@@ -1,262 +1,273 @@
-        "DefaultValue": "Yes",
-        "DisplaySettings": {
-          "Octopus.ControlType": "Select",
-          "Octopus.SelectOptions": "No|No\nYes|Yes"
-        }
-      },
-      {
-        "Id": "2a93a2e9-562f-4985-8933-acbf1b0e6df6",
-        "Name": "ChildProject.ManualIntervention.EnvironmentToUse",
-        "Label": "Approval Environment",
-        "HelpText": "*Optional*\n\nThe name of the environment you wish to pull the approvals from for the parent project.   It will look at all the deployments for the current release of the parent project and select the latest deployment to the specified environment.\n\nUsed when you are deploying to `Production` but want to pull the approvals from `Staging` or a `Prod Approval` environment.\n\nDefaults to the current environment name.",
-        "DefaultValue": "#{Octopus.Environment.Name}",
-        "DisplaySettings": {
-          "Octopus.ControlType": "SingleLineText"
-        }
-      },
-      {
-        "Id": "11101484-c68a-4a8b-aa4d-3f0a31e36ae2",
-        "Name": "ChildProject.ManualIntervention.Tenant.Name",
-        "Label": "Approval Tenant",
-        "HelpText": "*Optional*\n\nThe name of the tenant you wish to pull the approvals from for the parent project.   It will look at all the deployments for the current release of the parent project and select the latest deployment to the specified environment.\n\nUsed when you are deploying to `Production` but want to pull the approvals from `Staging` or a `Prod Approval` environment.\n\nDefaults to the current tenant name.  Will be skipped when doing a non-tenanted deployment.",
-        "DefaultValue": "#{Octopus.Deployment.Tenant.Name}",
-        "DisplaySettings": {
-          "Octopus.ControlType": "SingleLineText"
-        }
+    {
+      "Id": "d1d42293-e6fc-425e-a81c-973cc81eaa1d",
+      "Name": "ChildProject.WaitForFinish.Value",
+      "Label": "Wait for finish",
+      "HelpText": "Set to `Yes` to avoid waiting for the deployment to finish.  Will only be used when *What If* is set to `No`.",
+      "DefaultValue": "Yes",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "Yes|Yes\nNo|No"
       }
-    ],
-    "$Meta": {
-      "ExportedAt": "2022-06-20T13:40:09.112Z",
-      "OctopusVersion": "2022.1.2803",
-      "Type": "ActionTemplate"
     },
-    "LastModifiedBy": "bobjwalker",
-    "LastModifiedOn": "2022-06-20T13:40:09.112Z",
-    "Category": "octopus"
-  }
+    {
+      "Id": "52a65ba4-4ded-4f55-82e0-dc5a8c71a2f8",
+      "Name": "ChildProject.EnableEnhancedLogging.Value",
+      "Label": "Enable Enhanced Logging",
+      "HelpText": "Set to 'Yes' to retrieve output of the child release while waiting for the deployment to finish. Will only be used when *What If* is set to 'No' and *Wait for finish* is set to 'Yes'.",
```

Comment:
The literal values in the description of `Yes` and `No` should be wrapped in backticks for consistency with the descriptions of the other parameters.